chapters/02-filesystem.Rmd

```diff
@@ -1,16 +1,137 @@
-# Folder and file structure (RStudio R Projects)
+# Folder and file structure for R projects
 
-- Very basics of the shell (in RStudio),
+* Very basics of the shell (in RStudio),
 and filesystem terminology
-  - `cd`, `ls`, flags
-  - recursive
-  - paths: home and root
-- Working directory and how it is set with RStudio R Projects
-- Making use of the `fs` package for filesystem management
-  - similar functions in base R and why `fs` is preferred
-  - `dir_ls()`, `dir_tree()`
-  - `dir_map()`
-  - `file_delete()`, `file_info()`, `file_move()`
+  * `cd`, `ls`, flags
+  * recursive
+  * paths: home and root
+* Working directory and how it is set with RStudio R Projects
+* Making use of the `fs` package for filesystem management
+  * similar functions in base R and why `fs` is preferred
+  * `dir_ls()`, `dir_tree()`
+  * `dir_map()`
+  * `file_delete()`, `file_info()`, `file_move()`
+
+## Learning Objectives
+
+* access the shell through RStudio terminal to execute basic Unix commands
+* interpret paths as locations within folders
+* create RStudio projects and identify the working directory
+* apply functions from the `fs` package for filesystem management
+
+## Laying the foundation
+
+* text: This chapter serves two purposes:
+  * confirm everyone has the same baseline knowledge
+  * introduce foundational skills on which we'll build in subsequent chapters
+
+* text (and image?): referencing most recognizable panes of RStudio interface and their default locations
+  * Console: left side, where code is executed
+  * Environment: top right side, lists names of defined objects and information about them
+  * Files: lower right side, browser for navigating directories and files (other tabs in this pane include Plots, Packages, and Help)
+
+## The RStudio terminal
+
+* text (and image?): the RStudio terminal is a way to access the Unix shell from within RStudio. 
+  * Other methods of interacting with the shell include Terminal (Mac OSX), or Windows Subsystem for Linux (WSL) or Git-Bash (also Windows).
+  * RStudio terminal accesses these other methods via the RStudio interface: available via separate tab in Console
+  * Reference RStudio support materials?
+
+* text: why are we learning about the shell?
+  * some version control tasks can only be accomplished on the command line
+  * remote computing requires a remote login
+  * demonstrates and reinforces the importance of directory structure and project organization
+* text: accessing the terminal interface in RStudio
+  * locate tab in console window
+  * explain command prompt, and difference between bash and zsh (latter maybe not necessary given commands executed here?)
```

Comment:
The default is bash, yes? I think it might not be necessary to talk about zsh.

Reply:
The default for new Macs (e.g., within the last year) is zsh. I went and checked all commands in both bash and zsh and the output is identical, so I've updated the language!